### ISSUE TITLE
Style Guide -- EIP123 or EIP-123?

### ISSUE THREAD

**Issue by fulldecent** [ethereum/EIPs#1464]
Let us canonicalize the references for EIPs. They are inconsistently referred to as EIP-123 and EIP123 in the EIP documentation as well as the EIPs themselves. Another option is EIP 123. 

# Workplan

- [ ] Decide preferred citation format
- [ ] Update all text that is in the voice of the EIP authors (README, HTML files, EIP-1, etc.)

# Motivation

EIPs are much higher visibility than BIPs or PEPs. To reduce reader and community confusion, we should promote a consistent reference to these standards. 

It is unavoidable that end-users of Ethereum-based systems will be seeing EIP numbers. It will improve the professional image of these projects if we can have these look consistent.

In other words, seeing "You have 5 ERC-20 tokens" in MetaMask and "You have 5 ERC20 tokens" in Etherscan and "These are ERC 20 tokens" in the actual specification is less than optimal. 

# Background / data

EIP is inspired heavily from [BIPS](https://github.com/bitcoin/bips) which draws inspiration from [PEPS](https://github.com/python/peps). Both of those projects refer to proposals like BIP 1234.

Current usage of these words is tracked by Google. "ERC20" is a clear winner versus "ERC-20" and "ERC 20". 

https://trends.google.com/trends/explore?geo=US&q=%22erc%2020%22,%22erc-20%22,%22erc20%22

However this includes `ERC20`, the slug used in contracts.

**Comment by fulldecent** [comment]
Pinging esteemed top contributors to this repository for your thoughts please. @Arachnid @jamesray1 @cdetrio @axic @pirapira @chriseth @wanderer

**Comment by fulldecent** [comment]
I agree with all these points and do support ERC-20 naming for clarity. Previously I did not consider that ERC can be interpreted as a currency symbol. But with all the crazy currency symbols coming out this will now be my primary arguing point.

Let's not cite Wikipedia in this debate. I am an editor on that article, I think there is an XKCD for that. I stopped editing and came where when I realized this is an explicit decision we need to make before cleaning up external references.

**Comment by jamesray1** [comment]
OK, we agree then. Since you are actively involved with Ethereum it's probably best not to edit Wikipedia articles related to Ethereum. I've been told off for doing so.

**Comment by fulldecent** [comment]
👍 I am only working on the talk page, not editing actual pages. If editors can make an agreement here and it is authoritative (published to README.md, etc) then I will go back to Wikipedia talk page to point them to this authoritative source and my job is done.

**Comment by jamesray1** [comment]
They will probably want a secondary source. https://en.wikipedia.org/wiki/Wikipedia:No_original_research#Primary,_secondary_and_tertiary_sources.

**Comment by axic** [comment]
It seems I never left a comment here. To me it seems there is a tendency towards the `EIP-nnnn` format. It would be nice to codify it in EIP-1.

**Comment by fulldecent** [comment]
Related --> https://github.com/ethereum/EIPs/pull/2738

**Comment by axic** [comment]
This was actually accomplished by #2734.

![Screenshot](https://user-images.githubusercontent.com/20340/91622827-e4b74580-e990-11ea-9bee-f496b4bacd30.png)
